modules/nf-core/ensemblvep/main.nf

```diff
@@ -17,12 +17,9 @@ process ENSEMBLVEP {
     path  extra_files
 
     output:
-    tuple val(meta), path("*.ann.vcf")     , optional:true, emit: vcf
-    tuple val(meta), path("*.ann.tab")     , optional:true, emit: tab
-    tuple val(meta), path("*.ann.json")    , optional:true, emit: json
-    tuple val(meta), path("*.ann.vcf.gz")  , optional:true, emit: vcf_gz
-    tuple val(meta), path("*.ann.tab.gz")  , optional:true, emit: tab_gz
-    tuple val(meta), path("*.ann.json.gz") , optional:true, emit: json_gz
+    tuple val(meta), path("*.vcf.gz")  , optional:true, emit: vcf
```

Comment:
can we use the prefix in the output to catch just the proper vcf when we have already bgzipped ones as input?

Reply:
How do you mean exactly?

Reply:
similar to https://github.com/nf-core/modules/blob/5e34754d42cd2d5d248ca8673c0a53cdf5624905/modules/nf-core/snippy/run/main.nf#L15 for example

Reply:
Doesn't nextflow filter out all input files? Also, this is something to `stageAs` also fixes